packages/framework-provider-azure/src/helpers/query-helper.ts

```diff
@@ -18,10 +18,11 @@ export async function search(
   limit?: number | undefined,
   afterCursor?: Record<string, string> | undefined,
   paginatedVersion = false,
-  order?: SortFor<unknown>
+  order?: SortFor<unknown>,
+  projections = '*'
 ): Promise<Array<any> | ReadModelListResult<any>> {
```

Comment:
This method is used to perform any kind of query to any table in the database, so it's kind of coupling all user functions together: If a function requires a change in the `search` logic, then every other client function is at risk of stop working and, as developers, we need to take the extra care of making sure that the changes don't affect other user functions. Not needed for this PR, but I think it would be worth splitting this into separate use-case-specific query builder functions.